src/calculateRentalCost.js

```diff
@@ -29,6 +29,13 @@
 
 function calculateRentalCost(days) {
   // write code here
+  if (days >= 3 && days < 7) {
+    return (40 * days) - 20;
+  } else if (days >= 7) {
+    return (40 * days) - 50;
+  }
```

Comment:
50, 40 and 20 are the magic numbers, create variables for them

Reply:
You don't like magic?!

Reply:
oh sure, code isn't place for magic